hexrd/ui/calibration/polar_plot.py

```diff
@@ -151,3 +152,21 @@ def write_image(self, filename='polar_image.npz'):
             with h5py.File(filename, 'w') as f:
                 for key, value in data.items():
                     f.create_dataset(key, data=value)
+
+    def write_maud(self, filename='polar_to_maud.esg'):
+        filename = Path(filename)
+
+        with open(filename, 'w') as fid:
+            eta_vec = np.degrees(self.angular_grid[0])
+            intensities = self.raw_rescaled_img
```

Comment:
The reason SNIP and masks are not exported (as mentioned [here](https://github.com/HEXRD/hexrdgui/pull/1322#pullrequestreview-1268301956)) is because the `self.raw_rescaled_img` is being used here. This can be changed to `self.img` instead to include the background subtraction and masking.

I personally think it would be better to not have extra checkboxes pop up for whether to include the background subtraction/masking, and just export the polar image as-is. This is because the user can already turn off SNIP background subtraction in the polar view settings and turn off masking in the mask manager before exporting to Maud if they don't want to include them, and if they go this route, they have visual confirmation in the polar view of what they are actually exporting before it is done.

Still, if it is a common operation of users to visually see the SNIP background subtraction and masking in the polar view, but want to exclude them when exporting to Maud, we should add a pop-up with checkboxes.

Reply:
Hmm yes, I think this is a better idea.